cmd/legacy/offlinesync/offlinesync.go

```diff
@@ -29,29 +29,29 @@ func Run(v *viper.Viper) (int, error) {
 		var errauth api.ErrAuth
 		if errors.As(err, &errauth) {
 			return exitcode.ErrAuth, fmt.Errorf(
-				"failed to sync offline activity: %s. Find your api key from wakatime.com/settings/api-key",
+				"invalid api key... find yours at wakatime.com/settings/api-key. %s",
```

Comment:
We'll miss tracking where the error happened. The upstream won't be able to reference "offline" function in the logs.

Reply:
Doesn't it concat that together with `command failed: offline`?

Reply:
Oh you're right, it results in log:

```
"message":"failed to run command: invalid api key... find yours at wakatime.com/settings/api-key. authentication failed at...
```

I'll fix this in a new PR.